src/procedure/nodes/sequence.cpp

```diff
@@ -401,6 +401,11 @@ bool ProcedureNodeSequence::check() const
         // Check node branch if present
         if (node->branch() && !node->branch()->get().check())
             return false;
+
+        // Check validity of NodeValue-based keywords
+        for (auto &kwd : node->keywords().allOfType<NodeValue>())
+            if (!kwd->isValid())
+                return false;
```

Comment:
Just adding a couple of other `NodeValue`-based classes, and to note that these must reference the keyword class, not the datatype.  Code not tested, BTW!
```suggestion

        // Check validity of NodeValue-based keywords
        for (auto &kwd : node->keywords().allOfType<NodeValueKeyword>())
            if (!kwd->data().isValid())
                return false;
        for (auto &kwd : node->keywords().allOfType<Vec3NodeValueKeyword>())
            if (!kwd->data().x.isValid() || !kwd->data().y.isValid() || !kwd->data().z.isValid())
                return false;
        for (auto &kwd : node->keywords().allOfType<NodeValueEnumOptionsBaseKeyword>())
            if (!kwd->value().isValid())
                return false;
```

Reply:
Oops.. still familiarising myself with the `NodeValue` stuff.